packages/bot-web-ui/src/utils/bot-notifications.js

```diff
@@ -9,3 +12,24 @@ export const switch_account_notification = {
     type: 'warning',
     is_persistent: true,
 };
+
+export const journalError = clickFunction => {
+    return {
+        key: 'bot_error',
+        header: localize('The bot encountered an error while running.'),
+        message: (
+            <Button
+                className='notify__item-button'
+                onClick={clickFunction}
+                has_effect
+                type='button'
+                text={localize('View error in Journal')}
+                secondary
+            />
+        ),
+        type: 'danger',
+        is_persistent: true,
+        platform: ['DBot'],
```

Comment:
Could we define constants for our platforms somewhere and use those rather than passing in a string? e.g. in `deriv-app/packages/shared/src/utils/platform/platform.js`

Reply:
This was done. Thank you! :)